benchmarks/src/main/java/io/druid/benchmark/query/GroupByBenchmark.java

```diff
@@ -186,7 +199,7 @@ public void setup() throws IOException
     if (ComplexMetrics.getSerdeForType("hyperUnique") == null) {
       ComplexMetrics.registerSerde("hyperUnique", new HyperUniquesSerde(Hashing.murmur3_128()));
     }
-    executorService = Execs.multiThreaded(numSegments, "GroupByThreadPool");
+    executorService = Execs.multiThreaded(numProcessingThreads, "GroupByThreadPool[%d]");
```

Comment:
Most formats are with the thread enumeration after a dash rather than in brackets. Suggest keeping that here.